curator-framework/src/main/java/org/apache/curator/framework/api/RemoveWatchesBuilder.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.curator.framework.api;

```

Comment:
Internally, it looks like "local" mode acts only if the connection is lost. Maybe you can test the connection state when local is set and don't do retries, etc.